js/hotgraphicView.js

```diff
@@ -0,0 +1,148 @@
+define([
+    'core/js/adapt',
+    'core/js/views/componentDiffView',
+    './hotgraphicPopupView'
+], function(Adapt, ComponentDiffView, HotGraphicPopupView) {
+
+    var HotGraphicView = ComponentDiffView.extend({
+
+        renderAttributes: [
+            "displayTitle",
+            "body",
+            "title",
+            "instruction",
+            "_useGraphicsAsPins",
+            "_currentIndex",
+            "_items",
+            "_graphic",
+            "_isComplete",
+            "_canCycleThroughPagination"
```

Comment:
Is `_canCycleThroughPagination` meant to be on the popup’s view instead? Not sure if it makes any difference.